deploy.yml

```diff
@@ -558,41 +848,67 @@ jobs:
           title: Concourse Build $BUILD_ID
           title_link: http://concourse.dev.eq.ons.digital/builds/$BUILD_ID
 
-- name: Prod-Deployment-Plan
+- name: Author-PreProd-Deploy
+  public: true
   max_in_flight: 1
   plan:
   - get: eq-survey-runner-release
-    passed: [PreProd-Deploy]
+    passed: [build-eq-survey-runner]
+    params:
+      include_source_tarball: true
+    trigger: true
+  - get: eq-author-release
+    passed: [build-eq-author]
+    params:
+      include_source_tarball: true
+    trigger: true
+  - get: eq-author-api-release
+    passed: [build-eq-author-api]
+    params:
+      include_source_tarball: true
+    trigger: true
+  - get: eq-publisher-release
+    passed: [build-eq-publisher]
+    params:
+      include_source_tarball: true
+    trigger: true
+  - get: eq-schema-validator-release
+    passed: [build-eq-schema-validator]
+    params:
+      include_source_tarball: true
+    trigger: true
+  - get: go-launch-a-survey-release
+    passed: [build-go-launch-a-survey]
     params:
       include_source_tarball: true
     trigger: true
   - get: eq-terraform-release
-    passed: [PreProd-Deploy]
+    version: { tag: '19.0.0' }
     trigger: true
     params:
       include_source_tarball: true
   - get: eq-terraform-dynamodb-release
-    passed: [PreProd-Deploy]
+    version: { tag: 'v2.0' }
     trigger: true
     params:
       include_source_tarball: true
   - get: eq-terraform-ecs-release
-    passed: [PreProd-Deploy]
+    version: { tag: 'v4.0' }
     trigger: true
     params:
       include_source_tarball: true
   - get: eq-ecs-deploy-release
-    passed: [PreProd-Deploy]
+    version: { tag: 'v2.0' }
     trigger: true
     params:
       include_source_tarball: true
   - task: Deploy EQ VPC
     params:
-      TF_VAR_env: 'prod'
-      TF_VAR_aws_account_id: '((prod_aws_account_id))'
-      TF_VAR_aws_assume_role_arn: '((prod_aws_assume_role_arn))'
-      TF_VAR_vpc_cidr_block: '10.30.20.0/24'
-      TF_VAR_database_cidrs: ["10.30.20.96/28", "10.30.20.112/28", "10.30.20.128/28"]
+      TF_VAR_env: 'preprod'
+      TF_VAR_aws_account_id: '((preprod_aws_account_id))'
+      TF_VAR_aws_assume_role_arn: '((preprod_aws_assume_role_arn))'
+      TF_VAR_vpc_cidr_block: '10.30.21.0/24'
```

Comment:
Will Author ever need to peer with the corp vpc?
if not we can create a better bigger cidr block

Reply:
It may do at some point in the future (thinking ahead to a centralized auth service or interacting with some RAS/RM queue/service?), what would be the benefit of increasing the CIDR block and how might that affect us if we need to peer with corp VPC?

Reply:
If there is a plan to peer then it would be good to use a cidr block that doesn't conflict